build.rs

```diff
@@ -1,12 +1,12 @@
+use clap::Shell;
 use std::env;
 use std::fs::{self, File};
 use std::io::{self, Read, Write};
 use std::path::Path;
 use std::process;
 
-use clap::Shell;
-
 use app::{RGArg, RGArgKind};
+extern crate embed_resource;
```

Comment:
This line is [no longer needed](https://doc.rust-lang.org/nightly/edition-guide/rust-2018/path-changes.html) since Rust Edition 2018.

Reply:
Indeed, this should be removed.